.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
   pull_request:
+    paths-ignore:
```

Comment:
What if we instead ignore all changes to `*.rst` files only instead of entire `docs` tree? That way we won't miss any test coverage and won't need separate test-docs step, just docs-check step.

Reply:
I guess it doesn't work well when other modules have docs test and the commit contains only in docs directory. For instance, when we update `../../docs/src/main/sphinx/admin/resource-groups-example.json` without other changes, we still need to run tests for `trino-resource-group-managers` module. 

https://github.com/trinodb/trino/blob/master/plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/TestFileResourceGroupConfigurationManager.java#L201

Reply:
Yes, I think ignoring only *.rst files solves that issue since changes to `docs/src/main/sphinx/admin/resource-groups-example.json` will not cause CI to be skipped. CI will be skipped only for changes to `*.rst` files (which will be covered by docs.yml).

I think the existing impl is good. I'll spend some time later to see if my idea works or not.

Reply:
We shouldn't discern between .rst and .json or any other files under docs/ 
special handling on files types will be harder to follow and so "apparently unpredicable"